lms/static/js/dashboard/legacy.js

```diff
@@ -37,8 +37,6 @@
             notifications.focus();
         }
 
-        $('.action-more').bind('click', toggleCourseActionsDropdown);
```

Comment:
If none of this is used, let's break the removal out into a separate PR and get it merged. I have a feeling the new functionality will require a significant review.